src/main/java/cinema/model/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,21 @@
+package cinema.model.mapper;
+
+import cinema.model.CinemaHall;
+import cinema.model.dto.CinemaHallRequestDto;
+import cinema.model.dto.CinemaHallResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapper {
+    public CinemaHall getCinemaHallFromCinemaHallRequestDto(
+            CinemaHallRequestDto cinemaHallRequestDto) {
+        return new CinemaHall(cinemaHallRequestDto.getCapacity(),
+                cinemaHallRequestDto.getDescription());
```

Comment:
prefer setters over long constructors. Even if it takes more lines of code it's more readable and easier to refactor

Reply:
let's better remove such constructors and remove them from theirs classes